device-types/Raritan/PX3-5292R-M11.yaml

```diff
@@ -0,0 +1,58 @@
+---
+manufacturer: Raritan
+model: PX3-5292R-M11
+slug: px3-5292r-m11
```

Comment:
All slugs should have the manufacturer's name appended to them. This should start with `raritan-`

Reply:
Manufacturer name prepended